Java/core/src/main/java/com/amazon/randomcutforest/tree/AbstractCompactRandomCutTree.java

```diff
@@ -90,6 +99,15 @@ public AbstractCompactRandomCutTree(
         }
     }
 
+    public static boolean canUseSmallNodeStore(boolean float32precision, int capacity, int dimensions) {
```

Comment:
Can this take Precision instead of a boolean?

Reply:
Done.